examples/github-notification-filters/utils/getAllTags.ts

```diff
@@ -0,0 +1,23 @@
+import { Tag } from '@algolia/autocomplete-plugin-tags';
+
+import { NotificationFilter } from '../types';
+
+import { splitQuery } from '.';
+
+export function getAlltags(
```

Comment:
That's not really a util, it contains core logic of the example. We could inline it in `app.tsx`.

Reply:
True, addressed in 8895b48.